modules/disk-uploader/pkg/vmexport/vmexport.go

```diff
@@ -57,8 +60,18 @@ func WaitUntilVirtualMachineExportReady(client kubecli.KubevirtClient, namespace
 			return false, err
 		}
 
-		if vmExport.Status != nil && vmExport.Status.Phase == v1beta1.Ready {
-			return true, nil
+		if vmExport.Status != nil {
+			log.Logger().Info("VirtualMachineExport object status", zap.String("status", string(vmExport.Status.Phase)))
+
+			if vmExport.Status.Phase == v1beta1.Ready {
+				log.Logger().Info("VirtualMachineExport is in Ready state, and export source is not longer used")
+				return true, nil
+			}
+
+			if vmExport.Status.Phase == v1beta1.Pending {
+				log.Logger().Info("VirtualMachineExport is in Pending state, and export source is used")
+				return false, nil
```

Comment:
nit: It's okay to drop this line.

Reply:
I know, just wanted to keep consistency between if conditions. Let's leave it.